lib/helpers/webview.ts

```diff
@@ -357,6 +358,11 @@ async function cdpList(host: string, port: number): Promise<object[]> {
     await axios({
```

Comment:
this API can be extracted into a separate function call to avoid extra duplication

Reply:
Thank you! Extracted as 3e1ea66eee409e56519e5d1f885ec048dc66c1fe